docs/configuration/device-config/lora.mdx

```diff
@@ -63,9 +63,11 @@ In most cases you should use zero here. Units are in dBm.
 
 ### Bandwidth
 
-Bandwidth in MHz
+Bandwidth in kHz
 
-Certain bandwidth numbers are 'special' and will be converted by the device firmware to the appropriate floating point value: 31 -> 31.25MHz
+Certain bandwidth numbers are 'special' and will be converted by the device firmware to the appropriate floating point value: 31 -> 31.25kHz,
```

Comment:
Would you mind modifying this to use an unordered list or a table? I think that might help with the readability of this section.
```
- `31` -> `31.25kHz`
```
```
| Special Value/Reserved Value | Interpreted as |
| :---: | :---: |
| 31 | 31.25kHz |
```

Reply:
Done, please check if it's ok.